src/bindings.native.ts

```diff
@@ -28,11 +29,16 @@ export const KeyboardController = (
       )
 ) as KeyboardControllerModule;
 
+const KEYBOARD_CONTROLLER_NAMESPACE = "KeyboardController::";
 const eventEmitter = new NativeEventEmitter(KeyboardController);
 
 export const KeyboardEvents: KeyboardEventsModule = {
   addListener: (name, cb) =>
-    eventEmitter.addListener("KeyboardController::" + name, cb),
+    eventEmitter.addListener(KEYBOARD_CONTROLLER_NAMESPACE + name, cb),
+};
+export const FocusedInputEvents: FocusedInputEventsModule = {
```

Comment:
Let's add a JSDoc saying this is for internal usage only and is a subject to breaking changes without any warnings/announcement?